README.md

```diff
@@ -1 +1,36 @@
-# must-gather
\ No newline at end of file
+# must-gather for Open Data Hub
```

Comment:
Should we change references of Open Data Hub to RHODS?

Reply:
I think yes.